lib/ch/query.ex

```diff
@@ -185,9 +185,10 @@ defimpl DBConnection.Query, for: Ch.Query do
   defp encode_param(n) when is_integer(n), do: Integer.to_string(n)
   defp encode_param(f) when is_float(f), do: Float.to_string(f)
   defp encode_param(b) when is_binary(b), do: b
-  defp encode_param(b) when is_boolean(b), do: b
+  defp encode_param(true), do: "1"
+  defp encode_param(false), do: "0"
```

Comment:
Is it ok to use `1` and `0` in place of `true` and `false` in all cases?

Reply:
I reverted it now because I am not so sure